superset/views/core.py

```diff
@@ -1221,10 +1221,14 @@ def explore(self, datasource_type=None, datasource_id=None):
             )
 
         standalone = request.args.get("standalone") == "true"
+        user_roles = [role.name.lower() for role in get_user_roles()]
+        is_admin_role = "admin" in user_roles
```

Comment:
Is this role always called admin? Double checking to make sure it doesn't differ by deployment and need to rely on a config var

Reply:
yes. https://superset.incubator.apache.org/security.html#provided-roles